_pages/en/mandate.md

```diff
@@ -8,40 +8,52 @@ sections: Mission
 permalink: /mandate.html
 ---
 
-## Vision (image of the future)
+- Diagnose key organizational problems, waste, lead times and governance structures that inhibit the desired changes in behaviours
+- Recommend specific course of actions to governance bodies to adopt new methods of managing IT risks
+- Measure progress of adoption of strategies in IT teams and governance bodies as a performance indicator of the IT Strategy team
+- Challenge current thinking and approaches
 
-The [Canadian Experience](human-development-life-cycle.html) is when Canadians are benefiting seamlessly of the federal government services through modern technologies.
+<img src="assets/images/our-values.png" alt="">
```

Comment:
Buzz kill alert.....I'm pretty sure that using images instead of headers breaks accessibility rules (even with alt text). :-(

Reply:
Ok it seems like it might be possible to make image headers accessible but I'm not sure if this solution meets GC accessibility requirements.... https://www.webaxe.org/accessible-headers-with-graphics/

Reply:
As for images, it was just a try. If it breaks accessibility and GC rule I will return simple headers in markdown.  Just wanted to see the team's opinion on it. 
There is a way to do Graphic headers in HTML accessible not sure about markdown. I will wait for what the rest of the team will say otherwise will return H1.